data/brands/amenity/fuel.json

```diff
@@ -25,6 +25,16 @@
     }
   },
   "items": [
+    {
+     "displayName": "Astron Energy",
+     "locationSet": {"include": ["za"]},
+     "tags": {
+       "amenity": "fuel",
+       "brand": "Astron Energy",
+       "brand:wikidata": "Q120752181",
+       "name": "ASTRON ENERGY"
```

Comment:
From their store locator page, it looks like they call themselves "Astron Energy" in most places and on maps.  


<img width="800" alt="Screenshot 2023-07-24 at 9 52 51 AM" src="https://github.com/osmlab/name-suggestion-index/assets/38784/79247eb0-7520-4f7c-bafc-c86574bf5fac">

The allcaps "ASTRON ENERGY" name looks kind of weird (this is more normal in Japan but not in South Africa as far as I know).  Would you be ok with just using Title Case here?




```suggestion
       "name": "Astron Energy"
```

Reply:
Thanks @bhousel, for sure I'll update the name case.
Quick question about the 'name' tag - Does the name have to be what the logo of brand says or what it says on their map/store locator?
While adding 'Reel Cinemas' I got an impression it should be the logo, so used the same logic https://github.com/osmlab/name-suggestion-index/pull/8311#issuecomment-1607971655